webhook_server_container/libs/github_api.py

```diff
@@ -746,6 +795,20 @@ def process_pull_request_webhook_data(self):
             self.app.logger.info(f"{self.log_prefix} Creating welcome comment")
             self.pull_request.create_issue_comment(self.welcome_msg)
             self.create_issue_for_new_pull_request()
+
+            if self.jira_track_pr:
+                self.get_jira_conn()
+                if not self.jira_conn:
+                    self.app.logger.error(f"{self.log_prefix} Jira connection not found")
+                    return
+
+                self.app.logger.info(f"{self.log_prefix} Creating Jira story")
+                jira_story_key = self.jira_conn.create_story(
+                    title=self.issue_title,
+                    body=self.pull_request.url,
```

Comment:
this add the api url and not the pr html url
example:
https://api.github.com/repos/RedHatQE/ci-jobs-trigger/pulls/134
instead of
https://github.com/RedHatQE/ci-jobs-trigger/pull/134